### DISCUSSION TITLE
How can I sign the CLA?

### DISCUSSION THREAD

**Discussion by flaneur2020** [mindsdb/mindsdb#2926] [Get help]
Hello,

I've posted a PR about supporting clickhouse HTTP protocol (https://github.com/mindsdb/mindsdb/pull/2882), the CI suggested me that I needs pass MindsDB CLA Assistant first. But I haven't found related resources about how to sign the CLA in the contributing guide (https://github.com/mindsdb/mindsdb/blob/8bb8766b456855282c0f869f5e4625a96297a2bf/CONTRIBUTING.md). 

Any help about how can I pass the CLA Assistant?

Thank you in advance

**Comment by ZoranPandovski** [comment]
Hey @flaneur2020 it should trigger automatically but it looks like there is some issue with it. I just triggered it manually, feel free to sign it.

**Reply by flaneur2020** [reply]
Thank you a lot!

**Reply by bachng2017** [reply]
does it work for you @flaneur2020  ? We are having the same issue for https://github.com/mindsdb/mindsdb/pull/2928
I think there are  problems with current actions (CLA/tests). Seems that our PRs has nothing to do with the current failed test
```
...
Run if [ "$RUNNER_OS" == "Linux" ]; then

===============
test MySQL API
===============

Warning: Permanently added '3.220.66.106' (ECDSA) to the list of known hosts.
Load key "/home/runner/.ssh/db_machine": invalid format
ubuntu@3.220.66.106: Permission denied (publickey).
Found 0 MindsDB processes
Nothing to close
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.13/x[64](https://github.com/mindsdb/mindsdb/actions/runs/3075405380/jobs/4968795055#step:7:65)/lib/python3.7/site-packages/urllib3/connection.py", line 170, in _new_conn
    (self._dns_host, self.port), self.timeout, **extra_kw
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/urllib3/util/connection.py", line 96, in create_connection
    raise err
  File "/opt/hostedtoolcache/Python/3.7.13/x64/lib/python3.7/site-packages/urllib3/util/connection.py", line 86, in create_connection
    sock.connect(sa)
ConnectionRefusedError: [Errno 111] Connection refused
```

**Reply by ZoranPandovski** [reply]
Yes, this is a different issue. The CLA one should be resolved by upgrading the action (I hope) https://github.com/contributor-assistant/github-action/issues/103

**Reply by bilal-aamer** [reply]
> Hey @flaneur2020 it should trigger automatically but it looks like there is some issue with it. I just triggered it manually, feel free to sign it.

Can you please help guide me how to sign the CLA Assistant. Unable to get the relevant details.

**Reply by ZoranPandovski** [reply]
You just write `recheckcla` and the bot will send you the info
